python/src/GeneralLinearModelAlgorithm_doc.i.in

```diff
@@ -148,8 +148,13 @@ showing that :math:`\vect{\beta}^*` is a function of :math:`\vect{q}^*` only, an
 
 which leads to a further reduction of the log-likelihood function where both :math:`\vect{\beta}` and :math:`\sigma` are replaced by their expression in terms of :math:`\vect{q}`.
 
-The default optimizer is :class:`~openturns.TNC` and can be changed thanks to the *setOptimizationAlgorithm* method.
-User could also change the default optimization solver by setting the `GeneralLinearModelAlgorithm-DefaultOptimizationAlgorithm` resource map key to one of the :class:`~openturns.NLopt` solver names.
+The default optimizer is a :class:`~openturns.MultiStart` :class:`~openturns.TNC` with 20 starting points and can be changed with the :meth:`setOptimizationAlgorithm` method.
+User could also change the default optimization solver by setting:
+
+- the `GeneralLinearModelAlgorithm-DefaultOptimizationAlgorithm` resource map key to one of the :class:`~openturns.NLopt` solver names;
```

Comment:
that can be any of the solvers, not just Nlopt

Reply:
I did not know, this is from the current doc. Changing it now.

Reply:
```suggestion
- the `GeneralLinearModelAlgorithm-DefaultOptimizationAlgorithm` :class:`~openturns.ResourceMap` key to a solver name;
```